test_scripts/API/LastMileNavigation/GetWayPoints/008_GetWayPoints_IN_USE_one_app.lua

```diff
@@ -0,0 +1,102 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/25
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/embedded_navi/Get%20Destination_and_Waypoints.md
+-- Item: Use Case 1: Main Flow
+--
+-- Requirement summary:
+-- [GetWayPoints] As a mobile app I want to send a request to get the details of the destination 
+-- and waypoints set on the system so that I can get last mile connectivity.
+--
+-- Description:
+-- In case:
+-- 1) mobile application sends second valid and allowed request during the first one is processing on HMI
+-- SDL must:
+-- 1) respond "IN_USE, success:false" to mobile application
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonLastMileNavigation = require('test_scripts/API/LastMileNavigation/commonLastMileNavigation')
+
+local positiveReponse ={}
```

Comment:
@LitvinenkoIra Internal values of "positiveResponse" table can be defined directly